packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -1930,6 +1945,13 @@ class CupertinoTimerPicker extends StatefulWidget {
   /// Defaults to null, which disables background painting entirely.
   final Color? backgroundColor;
 
+  /// The uniform height of all children.
+  ///
+  /// All children will be given the [BoxConstraints] to match this exact
+  /// height. Must not be null and must be positive.
```

Comment:
```suggestion
  /// height. Must be a positive value.
```

No need to mention not being null: it's a non-nullable value.